apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -311,7 +311,10 @@ const DanceAiModal: React.FunctionComponent<DanceAiModalProps> = ({
     startGenerating();
   };
 
-  const handleStartOverClick = (usingHeader: boolean) => {
+  const handleStartOverClick = (
+    e: React.MouseEvent<HTMLDivElement, MouseEvent>
+  ) => {
+    const usingHeader = e.currentTarget.dataset.usingheader || false;
```

Comment:
I think I typed the event properly, but please double check my work.

Instead of taking `usingHeader` as a boolean, we attach it to the DOM element and read it out of there, finding the `currentTarget` of the event.